lib/idx/types/idx-js.ts

```diff
@@ -123,6 +157,37 @@ export interface IdxContext {
   };
 }
 
+export interface IdxContext {
```

Comment:
`IdxContext` is added twice. see https://github.com/okta/okta-auth-js/pull/1036/files#L95